### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by outoftardis**
# Description

Same change to README as in #1332 plus updated URL to community forum

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,7 +230,7 @@ This repository contains branches corresponding to both oneAPI and classical ver
...
 |-------|--------------|------|:-------------:|
-|oneDAL       |2021.1-beta10|[master](https://github.com/oneapi-src/oneDAL)</br>[rls/onedal-beta10-rls](https://github.com/oneapi-src/oneDAL/tree/rls/onedal-beta10-rls)|&nbsp;&nbsp;&nbsp;[Home page](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onedal.html)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[Documentation](http://oneapi-src.github.io/oneDAL/)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[System Requirements](https://software.intel.com/content/www/us/en/develop/articles/system-requirements-for-oneapi-data-analytics-library.html)|
+|oneDAL       |2021.1.3|[master](https://github.com/oneapi-src/oneDAL)</br>[rls/2021.1.3-rls](https://github.com/oneapi-src/oneDAL/tree/rls/2021.1.3-rls)|&nbsp;&nbsp;&nbsp;[Home page](https://software.intel.com/content/www/us/en/develop/tools/oneapi/components/onedal.html)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[Documentation](http://oneapi-src.github.io/oneDAL/)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[System Requirements](https://software.intel.com/content/www/us/en/develop/articles/system-requirements-for-oneapi-data-analytics-library.html)|
```
Comment (maria-Petrova) [inline README.md:233]: looks like the automation made these changes. The latest version should be changed to 2021.1 but we need to discuss it and merge it as a separate commit. Could you please revert these changes?

**Review by maria-Petrova — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (outoftardis) [reply README.md:233]: ok, sure

**Review by maria-Petrova — APPROVED** [review]
